ui/src/shared/utils/fromFlux.test.ts

```diff
@@ -209,4 +210,24 @@ there",5
     expect(() => fromFlux('\n\n\n\nyolo\ntrash\n\n\n\n')).not.toThrow()
     expect(fromFlux('\n\n\n\nyolo\ntrash\n\n\n\n').table.length).toEqual(0)
```

Comment:
you'd think that this test would check for leading and trailing whitespace interactions, but it doesn't